shell/browser/api/electron_api_cookies.cc

```diff
@@ -168,7 +188,24 @@ std::string InclusionStatusToString(
   if (status.HasExclusionReason(net::CanonicalCookie::CookieInclusionStatus::
                                     EXCLUDE_NONCOOKIEABLE_SCHEME))
     return "Cannot set cookie for current scheme";
-  return "Setting cookie failed";
+  LOG(WARNING) << "Cookie inclusion debug string: "
+               << status.GetDebugString() return "Setting cookie failed";
+}
+
+net::CookieSameSite StringToCookieSameSite(const std::string* str_ptr) {
```

Comment:
This should result in a JS exception being thrown if you typo the name, rather than silently defaulting to `no_restriction`

Reply:
Done